src/test/java/space/obminyashka/items_exchange/controller/AdvertisementControllerTest.java

```diff
@@ -102,4 +107,123 @@ void findPaginatedAsThumbnails_shouldReturnProperQuantityOfAdvertisementsThumbna
                 .andExpect(status().isOk())
                .andExpect(jsonPath("$.length()").value(advertisementRepository.findAll().size()));
     }
+
+    @Test
+    @WithMockUser(username = "admin")
+    void updateAdvertisement_shouldReturn400WhenNotValidAdvertisementSize() throws Exception {
+        AdvertisementDto existDtoForUpdate = AdvertisementDtoCreatingUtil
+                .createExistAdvertisementDtoForUpdateWithNotValidSize();
+
+        MvcResult mvcResult = mockMvc.perform(put("/adv")
+                .content(asJsonString(existDtoForUpdate))
+                .contentType(MediaType.APPLICATION_JSON)
+                .accept(MediaType.APPLICATION_JSON))
+                .andDo(print())
+                .andExpect(status().isBadRequest())
+                .andReturn();
+
+        assertTrue(mvcResult
+                .getResponse()
+                .getContentAsString()
+                .contains("must be between 1 and 50 symbols"));
```

Comment:
Not so good idea to hardcode the text - once the text will be changed the test would fail. It's better to get text of the same validation error and to compare with the result.
And you can do it in one line, it's not needed to be relined.

Reply:
ok, will be fixed